packages/collapsible/src/LionCollapsible.js

```diff
@@ -0,0 +1,175 @@
+    this._invokerNode.setAttribute('aria-controls', `collapsible-content-${uid}`);
+    this._contentNode.setAttribute('aria-labelledby', `collapsible-invoker-${uid}`);
+    this._contentNode.setAttribute('id', `collapsible-content-${uid}`);
+  }
+
+  /**
+   * Update aria labels on state change.
+   * @param {Object} changedProps - changed props
+   */
+  updated(changedProps) {
+    if (changedProps.has('opened')) {
+      this.__openedChanged();
+    }
+  }
+
+  disconnectedCallback() {
+    if (super.disconnectedCallback) {
+      super.disconnectedCallback();
+    }
+    this._invokerNode.removeEventListener('click', this.toggle);
+  }
+
+  /**
+   * Show extra content.
+   * @public
+   */
+  show() {
+    if (!this.opened) {
+      this.opened = true;
+    }
+  }
+
+  /**
+   * Hide extra content.
+   * @public
+   */
+  hide() {
+    if (this.opened) {
+      this.opened = false;
+    }
+  }
+
+  /**
+   * Toggle the current(opened/closed) state.
+   * @public
+   */
+  toggle() {
+    this.opened = !this.opened;
+    this.requestUpdate();
+  }
+
+  /**
+   * Show animation implementation in sub-classer.
+   * @protected
+   */
+  // eslint-disable-next-line class-methods-use-this, no-empty-function
+  async _showAnimation() {}
+
+  /**
+   * Hide animation implementation in sub-classer.
+   * @protected
+   */
+  // eslint-disable-next-line class-methods-use-this, no-empty-function
+  async _hideAnimation() {}
+
+  get _invokerNode() {
+    return Array.from(this.children).find(child => child.slot === 'invoker');
+  }
+
+  get _contentNode() {
+    return Array.from(this.children).find(child => child.slot === 'content');
+  }
+
+  get _contentHeight() {
+    const size = this._contentNode.getBoundingClientRect().height;
+    return `${size}px`;
+  }
+
+  /**
+   * Update content slot size and fire `opened-changed` event
+   * @private
+   */
+  __openedChanged() {
+    this.__updateContentSize();
+    this.__setCommonAriaAttributes();
+    this.dispatchEvent(new CustomEvent('opened-changed'));
+  }
+
+  /**
+   * Toggle extra content visibility on state change.
+   * @private
+   */
+  async __updateContentSize() {
+    if (this.opened) {
+      this._contentNode.style.setProperty('display', '');
+      await this._showAnimation({ contentNode: this._contentNode });
+    } else {
+      await this._hideAnimation({ contentNode: this._contentNode });
+      this._contentNode.style.setProperty('display', 'none');
+    }
+  }
+
+  /**
+   * Set default state for content based on `opened` attr
+   * @private
+   */
+  __setDefaultState() {
+    if (!this.opened) {
+      this._contentNode.style.setProperty('display', 'none');
+    }
+  }
+
+  /**
+   * Set common aria attributes for invoker and content nodes.
+   * @private
+   */
+  __setCommonAriaAttributes() {
+    this._invokerNode.setAttribute('aria-expanded', this.opened);
+    this._contentNode.setAttribute('aria-hidden', !this.opened);
```

Comment:
I mentioned it in private but just that we don't forget: probably don't need aria-hidden anymore ;)

Reply:
Yes, just removed it. 😅